examples/build.gradle

```diff
@@ -41,6 +63,22 @@ task geoserver(type: Exec) {
     dependsOn 'geoserverSleep'
 }
 
+task geoserverCleanup(type: Exec) {
+    description 'Run and wait for GeoServer with test data'
+    commandLine 'docker', 'exec', 'geoserver', 'rm', '-r', '/mnt/geoserver_datadir/logs'
+}
+
+task geoserverKillContainer(type: Exec) {
+    description 'Run and wait for GeoServer with test data'
+    commandLine 'docker', 'rm', '-f', 'geoserver'
```

Comment:
Shouldn't be `docker stop geoserver`? (the rm is already done because we have the `--rm`)

Reply:
It should, but I like to be sure...